examples/react-headless/src/components/Wallet.tsx

```diff
@@ -1,8 +1,17 @@
 import { FormEvent, memo, useCallback, useEffect, useState } from 'react';
-import { utils } from 'ethers';
 import { ApiPromise, WsProvider } from '@polkadot/api';
 import { BaseWallet, Account } from '@polkadot-onboard/core';
 
+const unitToPlanck = (units: string, decimals: number) => {
```

Comment:
What about leading zeros?!
e.x. unitToPlanck('0.01',5) output is 001000?!
I believe you need a simple utility to trim the leading zeros as well!